layouts/shortcodes/blocks/feature.html

```diff
@@ -10,5 +10,5 @@ <h4 class="h3">
 <div class="mb-0">
 {{ .Inner }}
 </div>
-{{ with .Get "url" }}<p><a href="{{ . }}">{{ with $url_text }}{{ $url_text }}{{ else }}{{ T "ui_read_more" }}{{ end }} …</a></p>{{ end }}
+{{ with .Get "url" }}<p><a href="{{ . }}">{{ with $url_text }}{{ $url_text }}{{ else }}{{ T "ui_read_more" }} …{{ end }}</a></p>{{ end }}
```

Comment:
Minor nit: for the sake of brevity:

```
{{ with $url_text }}{{ . }} ...
```

instead of

```
{{ with $url_text }}{{ $url_text }} ...
```
Another inconsistency: we use `{{ with .Get "url" }}`, I think we could use `{{ with .Get "url_text" }}` here, too (variable `$url_text` is used only once!).

And yes, I know that these refer to existing code, not to changes proposed from you. But why not take the chance while we are at it?

Reply:
Sure, good idea :) Thanks a lot for the comments!

Reply:
Checks are failing now 😢
I guess we need `$.Get "url_text"` here!?

Reply:
Ah, right. I shouldn't try changing code so late. I might be wrong, but my guess is that the problem is that  `{{ with .Get "url_text" }}` is within the context of `{{ with .Get "url" }}`. Don't know if there is an easy way around it.

Reply:
I think `$.Get` will do the trick